tests/test_event.py

```diff
@@ -46,3 +52,57 @@ def test_datesPresent(self):
         self.assertTrue(self.present_event.withinDays(-1, 1))
         self.assertFalse(self.present_event.future)
         self.assertFalse(self.present_event.past)
+
+    def test_datesStartsToday(self):
+        self.assertTrue(self.event_starts_today.starts_today)
+        self.assertFalse(self.event_starts_today.ends_today)
+
+    def test_datesEndsToday(self):
+        self.assertFalse(self.event_ends_today.starts_today)
+        self.assertTrue(self.event_ends_today.ends_today)
+
+    @classmethod
+    def createEventStartsToday(self, only_event=False):
+        id_string = "{}teststartstoday".format(datetime.datetime.now().year)
+        event = Event(
+            id=id_string,
+            end_date=datetime.datetime.today() + datetime.timedelta(days=2),
+            event_short="teststartstoday",
+            event_type_enum=EventType.REGIONAL,
+            first_eid="5561",
+            name="Test Event (Starts Today)",
+            start_date=datetime.datetime.today(),
+            year=datetime.datetime.now().year,
+            venue_address="123 Fake Street, California, USA",
+            website="http://www.google.com"
+        )
+        event = EventManipulator.createOrUpdate(event)
```

Comment:
I don't think you need lines 79-85 (and 102-108 below)

You should be able to get away with having this in `setUp()` (and one for `self.event_ends_today`)
The matches aren't needed because the test for starts_today and ends_today only depends on the Event object.

```
self.event_starts_today = Event(
            id=id_string,
            end_date=datetime.datetime.today() + datetime.timedelta(days=2),
            event_short="teststartstoday",
            event_type_enum=EventType.REGIONAL,
            first_eid="5561",
            name="Test Event (Starts Today)",
            start_date=datetime.datetime.today(),
            year=datetime.datetime.now().year,
            venue_address="123 Fake Street, California, USA",
            website="http://www.google.com"
        )
```